boards/arm/stm32f769i_disco/Kconfig.defconfig

```diff
@@ -12,6 +12,16 @@ config SPI_STM32_INTERRUPT
 	default y
 	depends on SPI
 
+config I2C
+	default y if KSCAN
+
+if KSCAN
+
+config KSCAN_FT5336
+	default y
+
+endif # KSCAN
```

Comment:
```suggestion
config KSCAN_FT5336
	default y if KSCAN
```

Reply:
@erwango Change done. Should I make another commit to exclude the board from this sample?.

Reply:
> @erwango Change done. 

Thanks

> Should I make another commit to exclude the board from this sample?.

Yes please. Though please take care to fix commit description: 
CI is failing due to incomplete commit message. It should looks like:
```
<path>: <component>: Title

Commit description.

Signed-off-by: <name> <email>
```
Cf https://docs.zephyrproject.org/latest/contribute/index.html#commit-guidelines

Reply:
Fixed the commit message and also opened a PR for the samples.